app/src/main/java/org/inspire/breath/activities/ChildAge.java

```diff
@@ -0,0 +1,42 @@
+package org.inspire.breath.activities;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+import org.inspire.breath.R;
+
+public class ChildAge extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_child_age);
+    }
+
+
+    public void Age_Click(View view){
```

Comment:
We already know the patient's age from the create profile screen, so you should be able to retrieve that value for the currently active patient and use that rather than prompting again